CHANGELOG.md

```diff
@@ -8,15 +8,6 @@ The format is based on [Keep a Changelog] and this project adheres to [Semantic
   [Semantic Versioning]: http://semver.org/spec/v2.0.0.html
 
 
-## [0.6.1] - 2023-09-16
```

Comment:
I think by mistake during rebase you removed this - sorry - it was part of a PR I merged only today :( I should have linked with what you were doing here.

Reply:
Ahh I didn't notice this in resolving the conflicts, will replace